src/SPECS/hootenanny.spec.in

```diff
@@ -312,16 +300,48 @@ EOT
 
     sudo sed -i s/password\:\ hoottest/password\:\ $DB_PASSWORD/ /var/lib/tomcat8/webapps/hoot-services/WEB-INF/classes/db/liquibase.properties
     sudo sed -i s/DB_PASSWORD=hoottest/DB_PASSWORD=$DB_PASSWORD/ /var/lib/tomcat8/webapps/hoot-services/WEB-INF/classes/db/db.properties
-    sudo sed -i s/dbPassword=hoottest/dbPassword=$DB_PASSWORD/g /var/lib/tomcat8/webapps/hoot-services/WEB-INF/classes/conf/hoot-services.conf
     sudo sed -i s/\<Password\>hoottest\<\\/Password\>/\<Password\>$DB_PASSWORD\<\\/Password\>/ /var/lib/tomcat8/webapps/hoot-services/WEB-INF/workspace/jdbc/WFS_Connection.xml
-    sudo sed -i s/\<jdbcPassword\>hoottest\<\\/jdbcPassword\>/\<jdbcPassword\>$DB_PASSWORD\<\\/jdbcPassword\>/ /var/lib/tomcat8/webapps/hoot-services/META-INF/maven/hoot/hoot-services/pom.xml
 
     # make sure tomcat is using correct Java
     sudo sed -i '/.*JAVA_HOME=.*/c\JAVA_HOME=\/usr\/java\/jdk1.8.0_111' /etc/tomcat8/tomcat8.conf
 
     sudo service tomcat8 restart
 }
 
+function migrateFiles() {
+    if [ ! -d "$HOOT_HOME/userfiles/ingest/processed" ]; then
+        mkdir -p $HOOT_HOME/userfiles/ingest/processed
+    fi
+
+    # tmp and upload will now reside under $HOOT_HOME/userfiles/
+    # The folders will be automatically created under $HOOT_HOME/userfiles by hoot-services webapp during Tomcat startup
+    rm -rf $HOOT_HOME/upload
+    rm -rf $HOOT_HOME/tmp
+
+    if [ -d "$HOOT_HOME/data/reports" ]; then
+        echo "Moving contents of $HOOT_HOME/data/reports to $HOOT_HOME/userfiles/"
+        cp -R $HOOT_HOME/data/reports $HOOT_HOME/userfiles/
+        rm -rf $HOOT_HOME/data/reports
+    fi
+
+    if [ -d "$HOOT_HOME/customscript" ]; then
+        echo "Moving contents of $HOOT_HOME/customscript to $HOOT_HOME/userfiles/"
+        cp -R $HOOT_HOME/customscript $HOOT_HOME/userfiles/
+        rm -rf $HOOT_HOME/customscript
+    fi
+
+    if [ -d "$HOOT_HOME/ingest" ]; then
+        echo "Moving contents of $HOOT_HOME/ingest to $HOOT_HOME/userfiles/"
+        cp -R $HOOT_HOME/ingest $HOOT_HOME/userfiles/
+        rm -rf $HOOT_HOME/ingest
+    fi
+
+    rm -rf $HOOT_HOME/data
```

Comment:
I was thinking this would go under the if block for data/reports, so it's kind of a one-time thing.  One danger with the current layout is if hoot repo or the user on a deployed system ever creates a data dir under $HOOT_HOME, this rpm post install is gonna whack it.  But that possibility is pretty remote I guess.